### DISCUSSION TITLE
Trying to return to E3 S1 Stock FW... printer crashes with "Invalid Mesh" now

### DISCUSSION THREAD

**Discussion by steve28** [mriscoc/Ender3V2S1#1005] [General]
I was trying out the professional firmware and I wanted to return to the stock Ender 3 S1 firmware:
Marlin2.0.8_Ender-3S1_HWv24S1_301_SWV3.1.9_Two_C_F401_FDM_LASER

I was able to flash it just fine, but the printer crashes everytime G28 or bed leveling is tried.  For example, if I try M420 A1, the error reported in the terminal is
```
Send: M420 S1
Recv: echo:Invalid mesh.
Recv: Error:Failed to enable Bed Leveling
```
I have tried reseting, storing config, re-flashing the firmware, etc.  Nothing I do helps.  It also crashes with this same error on G28, so basically I can't print.

Any ideas?

**Comment by Crazy-Charles** [comment]
Have you tried re-flashing back to the Pro firmware?

**Comment by androidand** [comment]
I have the same error, did you ever resolve this?
